src/WallGo/wallGoManager.py

```diff
@@ -71,13 +72,10 @@ class WallGoManager:
     details from the user.
     """
 
-    def __init__(self) -> None:
+    def __init__(self, config: Config) -> None:
         """"""
```

Comment:
IMO better to keep the old constructor and then do self.config = Config() in the constructor. So always build a default config first (OK because Config has defaults built in) and the user can then modify it later if needed. This way it's possible to have a minimal workflow where the user never has to explicitly create a Config manually.

Reply:
Ok that makes sense, I'll revert it.